kie-remote/kie-remote-jaxb-gen/pom.xml

```diff
@@ -292,11 +268,10 @@
               <goal>schemagen</goal>
             </goals>
             <configuration>
-              <includes>
-                <include>org/drools/**/*.java</include>
-                <include>org/jbpm/**/*.java</include>
-              </includes>
-              <verbose>true</verbose>
+              <sources>
+                <source>${project.basedir}/src/main/java/org/drools</source>
+                <source>${project.basedir}/src/main/java/org/jbpm</source>
+              </sources>
```

Comment:
@psiroky Pleaes keep `**/*.java`. The jars may eventually contain other classes besides `.java` classes, and I want to make sure that only `.java` classes are extracted into these directores. 

Good idea on adding the `${project.basedir}`!